### ISSUE TITLE
Improve the website (design / components)

### ISSUE THREAD

**Issue by Animeshz** [Animeshz/site#12]
The website is still under development & thus can accept various breaking changes right now. So you can tweak any part of the website if you think it is subjected for overall improvement (experience / ui / performance).

You can also add more components, but try to discuss it here before you do so. Some ideas can be view-count component.

Thanks for your contribution!

<sub>Sidenote: This website is based on Vitepress, that provides a way for extending it with [VueJS](https://vuejs.org/) components, current set of components are located in `src/theme` directory.</sub>

**Comment by sulavghimiree** [comment]
Please assign this to me.......

**Comment by CodeStrate** [comment]
@Animeshz I feel the light mode is superb. so is the dark mode but maybe the transition can be less jarring.

I'd love to contribute however I can to this project.

Let me know your thoughts.

**Comment by Animeshz** [comment]
Absolutely, @CodeStrate! The component design and CSS reside in the `src/theme` directory. Feel free to contribute whatever you'd like. If you have any questions or concerns, don't hesitate to reach out and ask!

**Comment by CodeStrate** [comment]
@Animeshz you may assign and I'll work on improvements.
if needed I can proof read the blogs for mistypings 
like in the Linux blog.

Kindly provide a shortlist for features you deem should be changed/added
I'll try to work and complete most if not all.

**Comment by Animeshz** [comment]
Since this is a general issue, this won't need an assignment, you (and others) may work on it multiple times.

Some ideas for new component as suggested above can be view counter, you may https://shields.io or any reliable badge provider to do that. You may think of more stuffs, like btn from any free external service (since we are on static pages - github.io), or things like share buttons (on social sites - one site / pr), and more things like that. Or you may check the colors improve contrast etc (I chose very randomly), grammatical errors or even any style issues if you think exists.

**Comment by CodeStrate** [comment]
So I can include badge and ui improvements under one PR correct?

**Comment by Animeshz** [comment]
Yes, but creating separate PR for separate purpose will be better!
